template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/appsettings.json

```diff
@@ -1,4 +1,36 @@
 ﻿{
+////#if (IndividualB2CAuth)
+//  "AzureAdB2C": {
+//    "Instance": "https:////login.microsoftonline.com/tfp",
+//    "ClientId": "11111111-1111-1111-11111111111111111",
+//    "CallbackPath": "/signin-oidc",
+//    "Domain": "qualified.domain.name",
+//    "SignUpSignInPolicyId": "MySignUpSignInPolicyId",
+//    "ResetPasswordPolicyId": "MyResetPasswordPolicyId",
+//    "EditProfilePolicyId": "MyEditProfilePolicyId"
+//  },
+////#elseif (OrganizationalAuth)
+//  "AzureAd": {
+//#if (MultiOrgAuth)
```

Comment:
@mlorbetske Tried to explain to me the rules for different levels of commenting in JSON, but they blew my brain, so I used flat indenting. Probably better for one of you guys to handle this.